lldb/source/Target/Platform.cpp

```diff
@@ -2076,6 +2076,12 @@ size_t Platform::GetSoftwareBreakpointTrapOpcode(Target &target,
     trap_opcode_size = sizeof(g_loongarch_opcode);
   } break;
 
+  case llvm::Triple::wasm32: {
+    static const uint8_t g_wasm_opcode[] = {0x00}; // unreachable
```

Comment:
Do we have a citation / prior art for this choice of encoding?

Reply:
It's part of the spec, I'll try to find a link to include here.